src/main/java/com/redhat/devtools/intellij/tektoncd/actions/OpenEditorAction.java

```diff
@@ -16,87 +16,44 @@
 import com.intellij.openapi.fileEditor.OpenFileDescriptor;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.ui.Messages;
+import com.intellij.openapi.util.Pair;
 import com.redhat.devtools.intellij.common.utils.UIHelper;
 import com.redhat.devtools.intellij.tektoncd.tkn.Tkn;
-import com.redhat.devtools.intellij.tektoncd.tree.ClusterTaskNode;
-import com.redhat.devtools.intellij.tektoncd.tree.ConditionNode;
-import com.redhat.devtools.intellij.tektoncd.tree.ClusterTriggerBindingNode;
-import com.redhat.devtools.intellij.tektoncd.tree.EventListenerNode;
-import com.redhat.devtools.intellij.tektoncd.tree.NamespaceNode;
-import com.redhat.devtools.intellij.tektoncd.tree.ParentableNode;
-import com.redhat.devtools.intellij.tektoncd.tree.PipelineNode;
-import com.redhat.devtools.intellij.tektoncd.tree.ResourceNode;
-import com.redhat.devtools.intellij.tektoncd.tree.TaskNode;
-import com.redhat.devtools.intellij.tektoncd.tree.TriggerBindingNode;
-import com.redhat.devtools.intellij.tektoncd.tree.TriggerTemplateNode;
+import com.redhat.devtools.intellij.tektoncd.tree.*;
```

Comment:
We don't use wildcard imports. I suggest you to disable intellij autoimport otherwise you will end up with this issue often https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
Yes, that may bad, I wasn't change default settings for wildcard imports, going to fix that